helix-term/src/main.rs

```diff
@@ -48,9 +48,53 @@ fn setup_logging(verbosity: u64) -> Result<()> {
 }
 
 pub struct Args {
+    display_help: bool,
+    display_version: bool,
+    verbosity: u64,
     files: Vec<PathBuf>,
 }
 
+fn parse_args(mut args: Args) -> Result<Args> {
+    let argv: Vec<String> = std::env::args().collect();
+    let mut iter = argv.iter();
+
+    iter.next(); // skip the program, we don't care about that
+
+    loop {
+        match iter.next() {
```

Comment:
This can be refactored into a `while let Some(arg) = iter.next()` loop for some more clarity and simpler matches :). It also will make it possible to match directly against the string literals instead of using `if`.

Reply:
Oh, that's neat, thanks, I've refactored the code as suggested